### PR TITLE
Assginment1:Kalyani

### PR DESCRIPTION
**Pull request by kalyani1942**

### CODE REVIEW

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        <h1>Assignment3_1</h1>
+    </title>
+</head>
+<body>  
+  
+    <h2>Ambersoft</h2>
+
+       <ol>
+    <li style="color: rgb(42, 157, 165);"> MI-MEAN</li>
+    <ol type = "i" style="color: rgb(13, 121, 106);">
```
Comment (sourabh-kanwade) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html]: please give proper indentation

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  
+     <li>MI-MEAN-9</li>
+     <ul type="disc">
+       <li>Anil</li>
+       <li>Manoj</li>
+     </ul>
+  </ol>
+  
+  
+
```
Comment (sourabh-kanwade) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html]: Remove space

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  
+    <ul type="disc">
+      <li>Triveni</li>
+      <li>Kalyani</li>
+    </ul>
+  </ol>
+  </ol>    
+  
+         
+
```
Comment (sourabh-kanwade) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html]: Remove Space

File: Kalyani/HTML/Assignment4/KalyaniA4_3.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        <td rowspan="3">Tuesday</td>
+        <td bgcolor="lightpink" style="text-align: center;">8:00am</td>
+        <td bgcolor="lightblue"style="text-align: center;">11:00am</td>
+        <td rowspan="1.5" style="text-align: center;">XPath</td>
+    </tr>
+    <tr>
+        <td  bgcolor="lightpink" style="text-align: center;">11:00am</td>
+        <td bgcolor="lightgreen" style="text-align: center;">2:00am</td>
+        <td rowspan="1.5" style="text-align: center;">XSL Transformations</td>
+    </tr>
```
Comment (sourabh-kanwade) [inline Kalyani/HTML/Assignment4/KalyaniA4_3.html]: Try to make it as given in Question.
maybe float values for rowspan doesn't work
try using higher number of rowspan eg. rowspan =6

**Review by sourabh-kanwade — CHANGES_REQUESTED** [review]

File: Kalyani/HTML/Assignment4/KalyaniA4_3.html (hunk 4 above)
Comment (kalyani1942) [reply Kalyani/HTML/Assignment4/KalyaniA4_3.html]: It's Work .The output for 1.5 is same as that of the output 3

Comment (sourabh-kanwade) [reply Kalyani/HTML/Assignment4/KalyaniA4_3.html]: decimal values for rowspan do not work
check your slack I have send screenshot of your code output on my machine.It does not look like what asked in question.

Comment (kalyani1942) [reply Kalyani/HTML/Assignment4/KalyaniA4_3.html]: i changed

Comment (kalyani1942) [reply Kalyani/HTML/Assignment4/KalyaniA4_3.html]: and changes committed now

**Review by sourabh-kanwade — CHANGES_REQUESTED** [review]
check slack

**Review by sourabh-kanwade — APPROVED** [review]

**Review by dayanandviveki — APPROVED** [review]
No Change

File: Kalyani/HTML/Assginment3/KalyaniA3_2main.html [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+            a:link {
+              color:red;
+            }
+            a:visited {
+              color: blue;
+            }
+          
+      </style>
+
+  </head>
```
Comment (Prashant698) [inline Kalyani/HTML/Assginment3/KalyaniA3_2main.html]: closed double head please remove one

File: Kalyani/HTML/Assginment3/KalyaniA3_2mi11.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+        color: #6fa6a0;
+      }
+      a:link {
+        color:red;
+      }
+      a:visited {
+        color: blue;
+      }
+    </style>
+  </head><body>
```
Comment (Prashant698) [inline Kalyani/HTML/Assginment3/KalyaniA3_2mi11.html]: please follow coading standard.

**Review by Prashant698 — COMMENTED** [review]
please follow coading standard

**Review by sourabh-kanwade — APPROVED** [review]

**Review by shivnanda6272 — APPROVED** [review]

**Review by somesh6797 — CHANGES_REQUESTED** [review]
Assignment 1 and 2 are not there only some empty files and remove space whenever necessary

**Review by dayanandviveki — APPROVED** [review]
No Changes Required

**Review by dayanandviveki — APPROVED** [review]
No Changes Required

**Review by shivnanda6272 — APPROVED** [review]

File: Kalyani/HTML/Assginment1/Assin.html [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assignment1</title>
+</head>
+<body>
+        <h1>HTML</h1>
+        <h3>Introduction</h3>
+        <p>HTML is a standard markup language for Web Pages.</p>
+    </body>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment1/Assin.html]: do proper indentation

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 8]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>
+        <h1>Assignment3_1</h1>
+    </title>
+</head>
+<body>  
+
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html]: remove extra lines

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+<html>
+<head>
+    <title>
+        <h1>Assignment3_1</h1>
+    </title>
+</head>
+<body>  
+  
+    <h2>Ambersoft</h2>
+
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html:13]: remove extra line

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+       <ol>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html]: indentation is not proper, do it proper

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+    <ol type = "i" style="color: rgb(13, 121, 106)">
+      <li>MI-MEAN-6</li>
+  
+      <ul type ="disc">
+      <li>Shweta</li>
+      <li>Ajay</li>
+      <li>Trupti</li>
+      </ul>
+      <li>MI-MEAN-7</li>
+      <ul type ="disc">
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html]: remove extra space

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+    </ul>
+    <li>SYSTEM-2</li>
+    <ul type="disc">
+      <li>Triveni</li>
+      <li>Kalyani</li>
+    </ul>
+  </ol>
+  </ol>    
+</body>
+</html>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html:57]: add new end line after </html> in all files

File: Kalyani/HTML/Assginment3/KalyaniA3_2contact.html [hunk 13]
```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Qust2</title>
+    <link rel="icon" type="image/x-icon" href="./assets/favicon.ico" />
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+<style>
+  body {
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_2contact.html]: indentation is very bad do check it in all files

File: Kalyani/HTML/Assginment3/KalyaniA3_2contact.html [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+        background-color: #3fe1f7;
+        color: #6fa6a0;
+      }
+      a:link {
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_2contact.html:13]: there should be always space after colon do this change in all files

File: Kalyani/HTML/Assginment3/KalyaniA3_2contact.html [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+        color:red;
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_2contact.html]: there should be always space after colon, do this change everywhere

File: Kalyani/HTML/Assginment3/KalyaniA3_2contact.html [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+      }
+      a:visited {
+        color: blue;
+      }
+    </style>
+</style>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_2contact.html]: why this extra style tag is there do remove it

File: Kalyani/HTML/Assginment3/KalyaniA3_2contact.html [hunk 17]
```diff
@@ -0,0 +1,61 @@
...
+  </head>
+<body>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_2contact.html:25]: do proper indentation

File: Kalyani/HTML/Assginment3/KalyaniA3_2contact.html [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+  <br />
+  <img
+    src="https://fixthephoto.com/blog/UserFiles/nature-photography-editing-after.jpg"
+    alt="Contact Us!!!"
+    style="float: left"
+    width="300px"
+  />
+  <div style="color: black;">
+    Contact us.
+    <p>contact us.Contact us.Contact us.Contact us.Contact us.Contact
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment3/KalyaniA3_2contact.html]: check the content is it proper?

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do proper indentation in all files, remove all extra linesand spaces

File: Kalyani/HTML/Assginment3/KalyaniA3_2contact.html (hunk 15 above)
Comment (kalyani1942) [reply Kalyani/HTML/Assginment3/KalyaniA3_2contact.html]: mam i have done all changes

File: Kalyani/HTML/Ass2/A2.html [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+<p>They called this search engine <i style="background-color: lightblue ">Backrub</i>.</p>
+    <h1>Initial public offering</h1>
+    <p>Google's initial public offering (IPO) took place five years later, on 19th August 2004.
+        <br> At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <ins>20</ins>
+        <br> years, until the year 2024.
+    </p>
+<center style="color: slategrey; font-size: 25px;">Thankyou!</center>
+</body>
+
+</html>
```
Comment (shivnanda6272) [inline Kalyani/HTML/Ass2/A2.html:36]: remove all extra lines. do proper indentation..

File: Kalyani/HTML/Ass2/A2.html [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+<p>Working from their dorm rooms, they built a search engine that used links to determine the importance of
+        <br> individual pages on the <b style="color: darkcyan;">World Wide Web</b>
+</p>    
+<p>They called this search engine <i style="background-color: lightblue ">Backrub</i>.</p>
+    <h1>Initial public offering</h1>
+    <p>Google's initial public offering (IPO) took place five years later, on 19th August 2004.
+        <br> At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <ins>20</ins>
+        <br> years, until the year 2024.
+    </p>
+<center style="color: slategrey; font-size: 25px;">Thankyou!</center>
```
Comment (shivnanda6272) [inline Kalyani/HTML/Ass2/A2.html:33]: don't use center as a tag.

File: Kalyani/HTML/Ass2/demo7.html [hunk 21]
```diff
@@ -0,0 +1,18 @@
...
+    a:visited{
+        color: green;
+    }
+    
+</style>
+</head>
+<body>
+    <a href="https://google.com">link text</a>
+</body>
+</html>
```
Comment (shivnanda6272) [inline Kalyani/HTML/Ass2/demo7.html:18]: add new line

File: Kalyani/HTML/Assginment1/Assin3.html [hunk 22]
```diff
@@ -0,0 +1,21 @@
...
+<body>
+    <h1>HTML</h1>
+    <h3>
+        <h3>Introduction</h3>
+        <p><u>HTML is a standard markup language for Web Pages</u></p>
+        <h2>HTML editor</h2>
+        <h3>Notepad or Textedit</h3>
+        <p>Web pages can be created and modified by using professional HTML editors. <br> However, for learning HTML we recommend a simple text editor like Notepad (PC) or TextEdit (Mac).</p>
+</body>
+
```
Comment (shivnanda6272) [inline Kalyani/HTML/Assginment1/Assin3.html:20]: remove extra lines...

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]
please make the changes according

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 23]
```diff
@@ -6,23 +6,20 @@ <h1>Assignment3_1</h1>
...
-      <li>MI-MEAN-6</li>
-  
-      <ul type ="disc">
-      <li>Shweta</li>
-      <li>Ajay</li>
-      <li>Trupti</li>
-      </ul>
+  <ol type = "i" style="color: rgb(13, 121, 106)">
+         <li>MI-MEAN-6</li>
+    <ul type ="disc">
```
Comment (shivnanda6272) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html]: do proper indentation...

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]

File: Kalyani/HTML/Assginment3/KalyaniA3_2contact.html [hunk 24]
```diff
@@ -29,33 +28,30 @@ <h1 style="color: black;">Ambersoft</h1>
...
+      us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+      us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact
+      us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+      us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact
+      us.Contact us.Contact us.Contact us.Contact us .Contact us.Contact
     </p>
-</div>
+  </div>
 </body>
 </html>
```
Comment (shivnanda6272) [inline Kalyani/HTML/Assginment3/KalyaniA3_2contact.html:48]: add new line.

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 25]
```diff
@@ -1,10 +1,11 @@
...
+<html lang="en">
 
 <head>
-    <title>
-        <h1>Assignment3_1</h1>
-    </title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <h1>Assignment3_1</h1>
```
Comment (shivnanda6272) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html]: where is <title> tag...

File: Kalyani/HTML/Assginment1/Assin.html [hunk 26]
```diff
@@ -11,7 +11,8 @@
 <body>
     <h1>HTML</h1>
     <h3>Introduction</h3>
-    <p>HTML is a standard markup language for Web Pages.</p>
+    <p>HTML is a standard markup language for Web Pages.
+    </p>
 </body>
```
Comment (shivnanda6272) [inline Kalyani/HTML/Assginment1/Assin.html:17]: remove extra lines. and new line at end

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]
do changes

File: Kalyani/HTML/Assginment1/Assin.html (hunk 26 above)
Comment (shivnanda6272) [inline Kalyani/HTML/Assginment1/Assin.html:17]: remove extra lines and add new line at end...

File: Kalyani/HTML/Ass2/A2.html [hunk 27]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (SnehaMakade) [inline Kalyani/HTML/Ass2/A2.html:1]: remove extra line and do proper indentation in all files

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do proper indentation in all files and remove extra lines

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**